tasty-query/shared/src/main/scala/tastyquery/reader/classfiles/JavaSignatures.scala

```diff
@@ -104,16 +108,25 @@ private[classfiles] object JavaSignatures:
     inline def readWhile[T](char: Char, inline op: T): List[T] =
       accumulate(accWhile = true, consume(char))(op)
 
-    def identifier(): SimpleName =
+    inline def charsWhile(inline f: Char @switch => Boolean): SimpleName =
```

Comment:
https://www.scala-lang.org/api/current/scala/annotation/switch.html

Reply:
I know what `@switch` does on a `match` ;) But it seems *very* weird to somehow assume that when put on the parameter type of a lambda type it will achieve anything remotely useful.

Reply:
I thought this would make it clear to pass a pattern match that compiles to a tableswitch

Reply:
Why would be `charsWhile` be concerned about that? What if I pass a function that performs two bounds checks or an array lookup? This is irrelevant here.